src/views/FfaListedView.vue

```diff
@@ -16,14 +16,14 @@
         v-show="selectedTab === listing"
         :ffaListing="ffaListing" />
       <button
```

Comment:
Purchase button should disable when drawer is open. See https://bulma.io/documentation/elements/button/ and scroll down to Disable